case_studies/glenaeon_rudolf_steiner_school/03_board_meeting/recap.md

```diff
@@ -13,7 +13,7 @@ One of the things I was asked to do was write a short proposal for our club.
 > We would like to start a student run code club within the school. It would meet
 > every Wednesday afternoon from 3:30-4:30 in the library. Ideally we would get
 > Chris to supervise the meetings. We would roughly follow the hackEDU
```

Comment:
As mentioned in my other comment, I decided to only change links in case studies. I'd love to hear your feedback on that.

The thought is that we shouldn't change other people's old writing.